src/main/java/org/scijava/ui/swing/script/autocompletion/JythonAutocompletionProvider.java

```diff
@@ -135,13 +136,13 @@ static public void removeAutoCompletionListener(final AutoCompletionListener lis
 		final Matcher m1f = fastImport.matcher(text);
 		if (m1f.find())
 			return asCompletionList(ClassUtil.findClassNamesForPackage(m1f.group(2)).map(formatter::singleToImportStatement), "");
-		
+
 		// E.g. "from ij.gui import Roi, Po" to expand to PolygonRoi, PointRoi for Jython
 		// or e.g. "importClass(Package.ij" to expand to a fully qualified class name for Javascript
 		final Matcher m2 = importStatement.matcher(text);
 		if (m2.find()) {
-			String packageName = m2.group(3),
-					 className = m2.group(4); // incomplete or empty, or multiple separated by commas with the last one incomplete or empty
+			final String packageName = m2.group(3); // incomplete or empty, or multiple separated by commas with the last one incomplete or empty
+			String className = m2.group(4);
```

Comment:
Not sure why this code was changed here? Seems like you have an opinionated code formatter. Now the comment is in the wrong place.

Reply:
Yes. My Eclipse has a will of its own. I hate it when it does that.  Will